connectors/connectors.sh

```diff
@@ -10,20 +10,15 @@ readonly VM_CONNECTORS_DATAPROC_DIR=/usr/local/share/google/dataproc/lib
 
```

Comment:
Let's remove "the Cloud Storage" on line number 3.

Reply:
Done.